school_hostel/models/hostel.py

```diff
@@ -107,7 +107,7 @@ def _compute_check_availability(self):
     student_ids = fields.One2many('hostel.student', 'room_id',
                                   string="Students")
 
-    _sql_constraints = [('room_no_unique', 'unique(room_no)',
+    _sql_constraints = [('room_no_unique', 'unique(room_no,name)',
```

Comment:
pep 8 mujab space avvi joie comma ma. This must be the basic style while we write code.